coreax/reduction.py

```diff
@@ -97,6 +100,9 @@ def __init__(
             refine_method, "refine_method", (coreax.refine.Refine, type(None))
         )
         self.refine_method = refine_method
+        """
+        Refine method
+        """
```

Comment:
Are these required by the automatic documentation generator? Could we instead document the attributes in the class docstring?

Reply:
Some of these docstrings already existed, so I tried to use what we had already. Agree that putting it in the docstring is probably preferable to having random multiline strings under each assignment in `__init__`, but I added a placeholder one for each that is referenced from elsewhere in the code in order to silence warnings. It's worth noting that most of them aren't sufficiently descriptive and it might be worth either fleshing them out in a future ticket (which could also reformat them), or removing them (and their corresponding references) entirely.

Reply:
Ok, lets leave them for a future ticket then.